bin/wazo-dist-upgrade-buster

```diff
@@ -147,6 +147,14 @@ migrate_systemd_system_conf() {
     fi
 }
 
+apt_get_install_wazo_dbms() {
+    if [ "$(dpkg-query --show wazo-dbms 2>/dev/null)" ]; then
```

Comment:
Suggestions:
* Use a function `is_installable`
* Use the return code of `dpkg-query` instead of `stdout` being empty